packages/tools/devtools/devtools-browser-extension/src/devtools/TelemetryLogging.ts

```diff
@@ -78,11 +79,17 @@ export class OneDSLogger implements ITelemetryBaseLogger {
 			httpXHROverride: fetchHttpXHROverride,
 		};
 
+		// NOTE: this doesn't really use environment variables at runtime.
+		// The dotenv-webpack plugin for webpack does a search-and-replace for `process.env.<variable-name>`
+		// and replaces them with inlined values if the corresponding variables exist in the environment
+		// at bundle time, or `undefined` if not.
+		const instrumentationKey = process.env.DEVTOOLS_TELEMETRY_TOKEN ?? "";
+
 		// Configure App insights core to send to collector
 		const coreConfig: IExtendedConfiguration = {
 			// A non-empty instrumentation key needs to provided for the logger to do anything.
 			// AB#5167 tracks how to inject one at build time to keep it out of source control.
```

Comment:
We can probably remove this comment line, since this PR is addressing it.

Reply:
Removed